SparkleSpin/Views/CustomTextField.swift

```diff
@@ -24,3 +25,15 @@ class CustomTextField: UITextField {
         tintColor = ThemeColor.Light.accentColorOne
     }
 }
+
+extension CustomTextField: UITextFieldDelegate {
+    
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        let characterLimit = 21
+        guard let text = text else { return false }
+        let currentTextString = text as NSString
```

Comment:
Is there a reason why we need to convert to `NSString` instead of using Swift `String`  methods like [this](https://developer.apple.com/documentation/swift/string/1641462-replacesubrange)? Is good to use Swift native types since the language is optimized for them.

Reply:
I agree! I had to do that here though because this delegate method uses `NSRange` and not `Range`. Also, `replaceSubrange(_:with:)` seems to be close to being deprecated, at least how it's indicated in XCode: 

<img width="791" alt="Screen Shot 2019-05-11 at 1 32 24 PM" src="https://user-images.githubusercontent.com/8409475/57573105-590f5380-73f1-11e9-8493-02ed2d6bb909.png">